types/react-native-dotenv/tsconfig.json

```diff
@@ -3,7 +3,7 @@
         "module": "commonjs",
         "lib": ["es6"],
         "esModuleInterop": true,
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
You keep pushing commits flipping the `convert-string` tsconfig, but it's the `react-native-dotenv` tsconfig I'm commenting on.